src/drivers/mct/bld/namelist_files/namelist_defaults_drv.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
+
+<?xml-stylesheet type="text/xsl" href="namelist_defaults.xsl"?>
+
+<namelist_defaults>
+
+<drv_flds_in_files>Buildconf/camconf/drv_flds_in,Buildconf/clmconf/drv_flds_in</drv_flds_in_files>
+
+</namelist_defaults>
```

Comment:
There should be no namelists_defaults.xml file. These values are contained in namelist_definition_drv.xml. If ALM is using this, it is incorrect. The prognostic components should not be using xml files in the driver cime_config/ directory. For CESM, CLM was modified (with just some slight changes) to not use this.